x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/index/RestrictedIndicesNames.java

```diff
@@ -14,12 +14,15 @@
 import java.util.Set;
 
 public final class RestrictedIndicesNames {
-    public static final String INTERNAL_SECURITY_INDEX_6 = ".security-6";
-    public static final String INTERNAL_SECURITY_INDEX_7 = ".security-7";
-    public static final String SECURITY_INDEX_NAME = ".security";
+    public static final String INTERNAL_SECURITY_MAIN_INDEX_6 = ".security-6";
+    public static final String INTERNAL_SECURITY_MAIN_INDEX_7 = ".security-7";
+    public static final String SECURITY_MAIN_ALIAS = ".security";
 
-    public static final Set<String> RESTRICTED_NAMES = Collections.unmodifiableSet(
-        Sets.newHashSet(SECURITY_INDEX_NAME, INTERNAL_SECURITY_INDEX_6, INTERNAL_SECURITY_INDEX_7));
+    public static final String INTERNAL_SECURITY_TOKENS_INDEX_7 = ".security-tokens-7";
```

Comment:
I don't really have an opinion. My _preference_ is to keep it as is, because we don't foresee such a necessity, and out of principle I am reluctant to design for feature we might need some day.
System indices, improvements to snapshots, might obsolete the tokens/ephemeral index before we need other ephemeral entities. My 2c, I am good with any alternative.